build/yaml/crd/nsx.vmware.com_subnetports.yaml

```diff
@@ -0,0 +1,110 @@
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.7.0
+  creationTimestamp: null
+  name: subnetports.nsx.vmware.com
+spec:
+  group: nsx.vmware.com
+  names:
+    kind: SubnetPort
+    listKind: SubnetPortList
+    plural: subnetports
+    singular: subnetport
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: SubnetPort is the Schema for the subnetports API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: SubnetPortSpec defines the desired state of SubnetPort
+            properties:
+              VirtualMachine:
+                description: VirtualMachine labels to realize SecurityPolices (VMSelector)
+                type: string
+              subnet:
+                description: Subnet defines the attachement of the SubnetPort
+                type: string
+              subnetset:
+                description: SubnetSet defines the attachement of the SubnetPort
+                type: string
+            type: object
+          status:
+            description: SubnetPortStatus defines the observed state of SubnetPort
+            properties:
+              conditions:
+                description: Conditions describes current state of SubnetPort
+                items:
+                  description: SubnetPortCondition defines the condition of policy.
+                  properties:
+                    message:
+                      description: Message shows a human readable message about the
+                        condition.
+                      type: string
+                    reason:
+                      description: Reason shows a brief reason of condition.
+                      type: string
+                    status:
+                      description: Status shows the status of condition, one of True
+                        or False.
+                      type: string
+                    type:
+                      description: Type defines the type of condition.
+                      type: string
+                  type: object
+                type: array
+              interfaceID:
```

Comment:
interfaceID should be the VIF ID instead of LSP ID, right?

Reply:
Yes you are right... Fixed.